modules/apps/transfer/keeper/keeper.go

```diff
@@ -156,3 +158,20 @@ func (k Keeper) AuthenticateCapability(ctx sdk.Context, cap *capabilitytypes.Cap
 func (k Keeper) ClaimCapability(ctx sdk.Context, cap *capabilitytypes.Capability, name string) error {
 	return k.scopedKeeper.ClaimCapability(ctx, cap, name)
 }
+
+// GetDenomHash retreives the hash key from the store for a demon trace
+func (k Keeper) GetDenomHash(ctx sdk.Context, denomTrace types.DenomTrace) (tmbytes.HexBytes, bool) {
```

Comment:
Why is this function necessary? This is the equivalent of:
```go
hash := denomTrace.Hash()
traceExists := k.HasDenomTrace(ctx, hash)
```

Reply:
@colin-axner @crodriguezvega  Do you mean to say the function is necessary but it would be better to implement it using k.HasDenomTrace?

Reply:
I would prefer not to add the function. I think the gRPC code should use `HasDenomTrace` instead

Reply:
@colin-axner @crodriguezvega  got it